unittests/ASM/VEX/bextr2.asm

```diff
@@ -0,0 +1,23 @@
+%ifdef CONFIG
```

Comment:
Drop this file, it duplicates the FEX_bug/ test I added for this today

Reply:
Oops, yep, added some ASM tests on accident.